Improve CronJob concept (#19270)
kubernetes/website: content/en/docs/concepts/workloads/controllers/cron-jobs.md

append 11 characters to the job name provided and there is a constraint that the
maximum length of a Job name is no more than 63 characters.

{{% /capture %}}
{{% capture body %}}

## CronJob

CronJobs are useful for creating periodic and recurring tasks, like running backups or
sending emails. CronJobs can also schedule individual tasks for a specific time, such as

> if you want to schedule a Job to happen when your cluster is likely to be idle.

Reviewer: ```suggestion
scheduling a Job for when your cluster is likely to be idle.
```
